### ISSUE TITLE
AttributeError: 'Win32Window' object has no attribute 'focus'

### ISSUE THREAD

**Issue by sunsikai** [asweigart/PyGetWindow#17]
![image](https://user-images.githubusercontent.com/33539903/66619003-e4de1380-ec0d-11e9-97f4-0dcf1a59495d.png)
A lot of methods fail, don't know why

**Comment by Ownmarc** [comment]
its called "activate" now

**Comment by JifengLegend** [comment]
Yes , it is called "activate", and I found the following methods:
win01.isActive
win01.activate()
gw.getActiveWindow()
